client/verta/tests/test_endpoint/test_endpoint.py

```diff
@@ -318,9 +318,9 @@ def test_update_autoscaling(self, client, created_endpoints, experiment_run, mod
         created_endpoints.append(endpoint)
 
         autoscaling = Autoscaling(min_replicas=0, max_replicas=2, min_scale=0.5, max_scale=2.0)
-        autoscaling.add_metric(CpuUtilization("0.5"))
-        autoscaling.add_metric(MemoryUtilization("0.7"))
-        autoscaling.add_metric(RequestsPerWorker("100"))
+        autoscaling.add_metric(CpuTarget("0.5"))
+        autoscaling.add_metric(MemoryTarget("0.7"))
+        autoscaling.add_metric(RequestsPerWorkerTarget("100"))
```

Comment:
Could we support letting the user pass in numbers (at least for these metrics), and we convert it to a `str` as a courtesy? The fact that the backend API wants `str`s is an implementation detail; accepting numbers in numerical form seems nicer for the user. And the backend will let us know if the value doesn't make sense.

...Also I'm feeling some déjà vu here, but I genuinely can't remember. Was there somewhere else I brought up this idea?
```suggestion
        autoscaling.add_metric(CpuTarget(0.5))
        autoscaling.add_metric(MemoryTarget(0.7))
        autoscaling.add_metric(RequestsPerWorkerTarget(100))
```

Reply:
Fixed.